specs/lib/processModelMaterialsCommonSpec.js

```diff
@@ -325,4 +325,84 @@ describe('processModelMaterialsCommon', function() {
         expect(techniqueVec4.parameters.joint.type).toEqual(WebGLConstants.FLOAT_VEC4);
         expect(techniqueMat3.parameters.joint.type).toEqual(WebGLConstants.FLOAT_MAT3);
     });
+
+    it('splits two materials with different attributes', function() {
```

Comment:
Small thing: I'm reading this spec title as "splitting two materials," like splitting two materials into four materials, instead of something like "splits a material used by meshes with different attributes."

Reply:
Fixed. I used your wording.